projects/js-packages/social-logos/tools/build

```
@@ -17,3 +17,13 @@ echo 'Starting full build...'
 ./tools/build-react &&
 echo 'Build complete.' ||
 echo 'Build failure!'
+
+if [[ $1 == '--prod' ]]; then
+	echo 'Checking for changed source files...'
+	if ! git diff --quiet './src'; then
```

May as well have it print out the changed files
```suggestion
	if ! git diff --exit-code --stat './src'; then
```